source/clear-linux/guides/maintenance/Telemetry-Enable-Disable.rst

```diff
@@ -0,0 +1,106 @@
+.. _telemtry-enable-disable:
+
+Enable and Disable Telemetry in Clear Linux
+###########################################
+
+|CLOSIA| includes a telemetry solution as part of the OS that records events
+of interest and reports them back to the development team via the telemetrics
+daemon, :command:`telemd`.  This functionality is maintained in the
+``telemetrics`` software bundle.
+
+.. Note::
+	The telemetrics functionality adheres to `Intel’s privacy policies`_
+	regarding the collection and use of Personally Identifiable Information
+	(PII) and is Open Source. Specifically, no intentionally identifiable
+	information about the user or system owner is collected.
+
+End users may enable or disable the telemetry component of |CL| or even
+redirect where records go if they wish to collect records for themselves.
+
+Install the telemetry software bundle
+*************************************
+
+During the initial installation of |CL| you are requested to join the
+Stability Enhancement Program and allow |CLOSIA| to collect anonymous reports
+to improve system stability.  If you chose not to join this program at that
+time then the telemetry software bundle was not added to your system.
+
+To install the telemetry bundle, enter the following command as either the
+root user or with :command:`sudo` priveleges:
+
+.. code-block:: console
+
+	sudo swupd bundle-add telemetrics
+
+This will add the telemetrics-client to your system and you will automatically
+be opted-in for the service.
+
+Enable Telemetry
+****************
+
+To start telemetry on your system run the following command:
+
+.. code-block:: console
+
+	sudo telemctl start
+
+This will enable and start the :command:`telemd` daemon and your system will
+begin to send telemetry data to the server defined in the file
+:file:`/etc/telemetrics/telemetrics.conf` and if this file does not exist it
+will use the file :file:`/usr/share/defaults/telemetrics/telemetrics.conf`.
+
+Disable Telemetry
+*****************
+
+To disable the telemetry daemon run the following command:
+
+.. code-block:: console
+
+	sudo telemctl stop
+
+Opt-out of telemetry
+********************
+
+If you no longer want to send telemetrics data from your system, you can
+opt-out of the service by running the command:
+
+.. code-block:: console
+
+	sudo telemctl stop
+	sudo telemctl opt-out
+
+This will create the file :file:`/etc/telemetrics/opt-out` and stop the
+telemetry services.
+
+Opt-in to telemetry
+*******************
+
+Conversely, to opt-in to the telemetry services, simple enter the opt-in
+command and start the service:
+
+.. code-block:: console
+
+	sudo telemctl opt-in
+	sudo telemctl start

```

Comment:
Similarly here, `telemctl opt-in` will start services for you after opting in to telemetry. This behavior is arguably more confusing. For example, if the user wants to opt-in and *not* immediately start telemetry services, this is not currently possible with telemctl.

Reply:
@phmccarty , I could add a section that they can opt-in and then immediately do a telemctl stop.  They would still be opted-in but they could control the functionality of the service.  For instance if they wanted to add their own custom telemetry to a program being developed they can start and stop it from the cmdline.  I did remove the telemctl start since it isn't required.

Reply:
@tpleavitt Yes, I think this info would be helpful in a separate section. If users want to opt in, but not run services right away, running `telemctl opt-in` followed by `telemctl stop` is a good approach for now.